jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/DroolsDeclaredTypeSequenceFlowExpressionTest.java

```diff
@@ -0,0 +1,60 @@
+package org.jbpm.bpmn2;
```

Comment:
@mswiderski @DuncanDoyle Please add the missing header asap. We need to merge the fix in order to have a chance to include this in 6.5.0.Final